### ISSUE TITLE
Batch upload broken

### ISSUE THREAD

**Issue by Ma-ryan** [myimouto/myimouto#131]
Attempting to batch upload has all jobs stuck at "pending" status with no progress at all. Also, I'm not sure if it's relevant, but ``upload_batch_posts`` at job tasks complains of an error and gives me this message: ``Rails\ActiveRecord\Exception\RuntimeException: Can't write unknown property 'is_upload' for model Post``

Any ideas for a fix?

**Comment by Ma-ryan** [comment]
Any update to this issue?

**Comment by Parziphal** [comment]
Uh sorry, I completely forgot about this.
Check if that update fixes things.

**Comment by Kille250** [comment]
upload pictures via url isn't generating or saving the png file.

**Comment by Ma-ryan** [comment]
Nope. Uploads are still stuck at pending.

**Comment by Parziphal** [comment]
Is any error logged?

**Comment by Ma-ryan** [comment]
Unfortunately, no errors have been logged anywhere. Here is the terminal output of the job task script:

```
*** PHP Notice in Tag#get_current_history ***
Trying to get property of non-object 
#0 lib/Moebooru/Versioning/VersioningTrait.php(62): Rails::errorHandler(8, 'Trying to get p...', '/home/testuser/m...', 62, Array)
#1 lib/Moebooru/Versioning/VersioningTrait.php(107): Tag->get_current_history()
#2 Rails/ActiveRecord/Connection.php(207): Tag->Moebooru\Versioning\{closure}()
#3 Rails/ActiveRecord/Base.php(158): Rails\ActiveRecord\Connection->transaction(Object(Closure), NULL)
#4 lib/Moebooru/Versioning/VersioningTrait.php(117): Rails\ActiveRecord\Base::transaction(Object(Closure))
#5 Rails/ActiveRecord/Base.php(652): Tag->save_versioned_attributes()
#6 Rails/ActiveRecord/Base.php(861): Rails\ActiveRecord\Base->runCallbacks('save', Object(Closure))
#7 Rails/ActiveRecord/Base.php(62): Rails\ActiveRecord\Base->_create_do()
#8 app/models/Tag.php(53): Rails\ActiveRecord\Base::create(Array)
#9 app/models/Post.php(310): Tag::find_or_create_by_name('tagme')
#10 Rails/ActiveRecord/Base.php(652): Post->before_creation()
#11 Rails/ActiveRecord/Base.php(785): Rails\ActiveRecord\Base->runCallbacks('validation_on_c...', Object(Closure))
#12 Rails/ActiveRecord/Base.php(790): Rails\ActiveRecord\Base->_validate_data('create')
#13 Rails/ActiveRecord/Base.php(62): Rails\ActiveRecord\Base->_create_do()
#14 app/models/BatchUpload.php(34): Rails\ActiveRecord\Base::create(Array)
#15 app/models/JobTask.php(239): BatchUpload->run()
#16 app/models/JobTask.php(112): JobTask->execute_upload_batch_posts()
#17 app/models/JobTask.php(9): JobTask->execute()
#18 script/daemons/job_task_processor.php(6): JobTask::execute_once()
#19 {main}
Error context:
```

**Comment by Parziphal** [comment]
Don't know if you're still using this, but this error was just fixed with 61b48d0 (I forgot to reference this issue). It was caused due to a bug in the framework.
